loadtest/setup_server.go

```diff
@@ -247,15 +247,55 @@ func checkConfigForLoadtests(adminClient *model.Client4) error {
 
 	mlog.Info("EnableIncomingWebhooks is true")
 
-	if *serverConfig.ServiceSettings.DEPRECATED_DO_NOT_USE_EnableOnlyAdminIntegrations {
-		mlog.Info("Disabling only admin integrations for loadtest.")
-		*serverConfig.ServiceSettings.DEPRECATED_DO_NOT_USE_EnableOnlyAdminIntegrations = false
-		if _, resp := adminClient.UpdateConfig(serverConfig); resp.Error != nil {
-			mlog.Error("Failed to set EnableOnlyAdminIntegrations", mlog.Err(resp.Error))
+	mlog.Info("Disabling only admin integrations for loadtest. 1/2")
+	if role, resp := adminClient.GetRoleByName(model.TEAM_USER_ROLE_ID); resp.Error != nil {
+		mlog.Error("Failed to get role", mlog.String("role", model.TEAM_USER_ROLE_ID), mlog.Err(resp.Error))
+		return resp.Error
+	} else {
+		newPermission := []string{}
+		newPermission = append(newPermission, role.Permissions...)
+	OUTER:
```

Comment:
I don't love this kind of construct, let me think about it

Reply:
@jespino - you don't like the OUTER for? I can rewrite that... Or do you have an issue with the whole approach of role patching?

Reply:
Sorry, I don't like the "goto" approach, I not necessarily against it and I haven't a proper reason against it but I don't like it.

Reply:
It seems like this is only necessary because the server doesn't deduplicate incoming permissions on patching a role. Maybe we can just fix that in the server and unconditionally append the new permissions instead?